src/components/views/elements/AppTile.tsx

```diff
@@ -406,6 +409,11 @@ export default class AppTile extends React.Component<IProps, IState> {
             ? Container.Right
             : Container.Center;
```

Comment:
Please indent the 3 lines below `targetContainer =` by one step. A bit surprised linting did not yell about this...